### DISCUSSION TITLE
Pixi works really well 😍

### DISCUSSION THREAD

**Discussion by jonashaag** [prefix-dev/pixi#286] [Show and tell]
I just wanted to say thank you for this awesome tool. It works really well for anything I've done with it so far, with zero problems and no weird bugs.

I've replaced almost all Conda/Mamba/Micromamba usage with Pixi on my local machine. It looks like Pixi will obsolete Micromamba and Mamba for local workflows very soon.

Keep up the great work!

**Comment by baszalmstra** [comment]
Thanks for the kind words @jonashaag ! Good to hear that your enjoying it!
